eng/dogfood.ps1

```diff
@@ -39,7 +39,7 @@ try {
 
   $TestDotnetRoot = Join-Path $ArtifactsDir "bin\redist\$configuration\dotnet"
 
-  $testDotnetVersion = (Get-Childitem -Directory "$TestDotnetRoot\sdk")[-1]
+  $testDotnetVersion = (Get-Childitem -Directory "$TestDotnetRoot\sdk")[-1].Name
   $env:DOTNET_MSBUILD_SDK_RESOLVER_SDKS_DIR = Join-Path $TestDotnetRoot "sdk\$testDotnetVersion\Sdks"
```

Comment:
If it's returning a full path, can we instead do:
```suggestion
  $testDotnetVersion = (Get-Childitem -Directory "$TestDotnetRoot\sdk")[-1]
  $env:DOTNET_MSBUILD_SDK_RESOLVER_SDKS_DIR = "$testDotnetVersion\Sdks"
```

? Seems a bit cleaner to me

Reply:
After my testing, I think this is actually a bad idea. If we assume it's a full path, and it sometimes isn't, we'll fail.

Reply:
It returns a full path on `pwsh` only, not on `powershell`, so that wouldn't work on both.